app/routes/events_.$eventId.tsx

```diff
@@ -102,3 +102,12 @@ export function ErrorBoundary() {
   const { eventId } = useParams();
   return <div className="error-container">There was an error loading event by the id {eventId}.</div>;
 }
+
+export const meta: MetaFunction = ({ data }) => {
+  const eventFullName = data.event.name;
```

Comment:
Check out this section in the Remix docs: https://remix.run/docs/en/main/route/meta#data. 

You can use the `loader` function to infer the type of data!